scripts/rca-demo/post-setup

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+SCRIPTS_DIR=$(dirname "${BASH_SOURCE[0]}")
+source $SCRIPTS_DIR/shared.sh
+source $SCRIPTS_DIR/.env
+
+create_data_view () {
+	ID=$1
+	INDICES=$2
+
+	EXISTS=$(curl -s $KIBANA_URL/api/data_views/data_view/$ID -H "Authorization: ApiKey $OTEL_DEMO_ES_API_KEY" -H 'elastic-api-version: 2023-10-31')
+
+	if [[ $EXISTS == '{"statusCode":404,"error":"Not Found"'* ]]; then
+		echo "Creating data view $1 for indices $2"
+		curl -X POST $KIBANA_URL/api/data_views/data_view -H 'kbn-xsrf: bananas' -H 'Content-Type: application/json' -H 'elastic-api-version: 2023-10-31' -H "Authorization: ApiKey $OTEL_DEMO_ES_API_KEY" -d "{\"data_view\": { \"name\": \"$1 (Automated by Demo CLI)\", \"title\": \"$2\", \"id\": \"$1\", \"timeFieldName\": \"@timestamp\" }}"
+	else
+		echo "Data view $1 already exists"
+	fi
+}
+
+create_rule () {
+	ID=$1
+	DATA_FILE=$2
+
+	curl -X POST $KIBANA_URL/api/alerting/rule/$ID -H 'kbn-xsrf: bananas' -H "Authorization: ApiKey $OTEL_DEMO_ES_API_KEY" -H 'Content-Type: application/json; elastic-api-version: 2023-10-31' -d @$SCRIPTS_DIR/data/$DATA_FILE
+
+	echo ""
+}
+
+echo "Setting up Kibana assets for the demo..."
+
+# Load local user .env file
+ENV_FILE="./scripts/rca-demo/.env"
+if [ -f $ENV_FILE ]; then
+	echo "Sourcing env vars from $ENV_FILE ..."
+  source $ENV_FILE
+fi
+
+if [ -z "${KIBANA_URL}" ]; then
+  die "You must set KIBANA_URL so that we can post API requests there (please include base path if one exists)"
+fi
+
+if [ -z "${OTEL_DEMO_ES_API_KEY}" ]; then
+  die "You must set OTEL_DEMO_ES_API_KEY (must be valid API key for ES and Kibana APIs)"
+fi
+
+# Create data views
+create_data_view otel_logs_data logs-*otel* 
+
+# Create rules
+create_rule 9055220c-8fb1-4f9f-be7c-0a33eb2bafc5 rule-01.json
+
+# Update mappints in ES
+curl -X PUT http://localhost:9200/logs-*otel*/_mapping -H "Authorization: ApiKey $OTEL_DEMO_ES_API_KEY" -H "Content-Type: application/json" -d @$SCRIPTS_DIR/data/mapping-message-alias.json
```

Comment:
this will fail if the data stream doesn't exist, in that case you need to update the component template as well.

Reply:
Thanks, I'll add the component update in a follow up and just make this fail silently